Lib/optparse.py

```diff
@@ -254,6 +253,7 @@ def _format_text(self, text):
         """
         text_width = max(self.width - self.current_indent, 11)
         indent = " "*self.current_indent
+        import textwrap
```

Comment:
Style: move local imports to the top of the block
```suggestion
        import textwrap
        text_width = max(self.width - self.current_indent, 11)
        indent = " "*self.current_indent
```

Reply:
Sure, makes sense!

For what it's worth the reason I did it this way was a direct consequence of having just been reading argparse.py, which breaks this style suggestion, while documenting the motivation of the change. :) I suppose it is not worth bothering with the churn there as it's just a style nit on code that isn't being modified already...